### PR TITLE
OpenAIAssistantRunnable fix

### PR DESCRIPTION
**Pull request by SmittieC**
The invoke code was copied from [here](https://github.com/langchain-ai/langchain/blob/54adcd9e828e24bb24b2055f410137aca6a12834/libs/langchain/langchain/agents/openai_assistant/base.py#L256) and instead of passing `file_ids`, we now pass in `attachments`

### CODE REVIEW

File: apps/service_providers/llm_service/main.py [hunk 1]
```diff
@@ -2,18 +2,83 @@
...
+                tool_outputs = self._parse_intermediate_steps(input["intermediate_steps"])
+                run = self.client.beta.threads.runs.submit_tool_outputs(**tool_outputs)
+            # Starting a new thread and a new run.
+            elif "thread_id" not in input:
+                thread = {
+                    "messages": [
+                        {
+                            "role": "user",
+                            "content": input["content"],
+                            "attachments": input.get("attachments", {}),
```
Comment (SmittieC) [inline apps/service_providers/llm_service/main.py:47]: previously this was `file_ids`

File: apps/service_providers/llm_service/runnables.py [hunk 2]
```diff
@@ -280,6 +280,24 @@ def invoke(self, input: str, config: RunnableConfig | None = None) -> ChainOutpu
...
+                    file_resources[file.external_id].append(resource)
+                else:
+                    file_resources[file.external_id] = [resource.tool_type]
+
+        for file_id, resources in file_resources.items():
+            tools = [{"type": resource} for resource in resources]
+            attachment = {"file_id": file_id, "tools": tools}
+            attachments.append(attachment)
+
+        input_dict["attachments"] = attachments
```
Comment (snopoke) [inline apps/service_providers/llm_service/runnables.py:300]: This isn't necessary. The attachments here are per-message attachments which we don't support.

**Review by snopoke — CHANGES_REQUESTED** [review]
One comment but otherwise looks good.

File: apps/service_providers/llm_service/runnables.py (hunk 2 above)
Comment (SmittieC) [reply apps/service_providers/llm_service/runnables.py:300]: Of course, we don't have to pass it along with each message. I wasn't thinking straight. Thanks!

**Review by snopoke — APPROVED** [review]

**Pull request merged**